partner_compassion/models/product.py

```diff
@@ -7,13 +7,33 @@
 #    The licence is in the file __manifest__.py
 #
 ##############################################################################
-from odoo import models, fields
+from odoo import api, models, fields
+from odoo.exceptions import ValidationError
+
+
+def check_fund_id(record):
```

Comment:
Why doing an external public method instead of coding it directly inside the `_check_fund_id` method?

Reply:
Thought it would be a good idea to have this method public for other models, but actually no other model has an attribute fund_id. Will write everything under _check_fund_id.